include/xrpl/protocol/detail/features.macro

```diff
@@ -29,6 +29,8 @@
 // If you add an amendment here, then do not forget to increment `numFeatures`
 // in include/xrpl/protocol/Feature.h.
 
+// fix1782: Check flags in Credential transactions
+XRPL_FIX    (1782,                       Supported::yes, VoteBehavior::DefaultNo)
```

Comment:
I think it would make sense to use `DefaultYes` here.

Reply:
changed to Yes